sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -365,7 +421,9 @@ class SessionCatalog(
   def lookupRelation(name: TableIdentifier, alias: Option[String] = None): LogicalPlan = {
     synchronized {
       val db = formatDatabaseName(name.database.getOrElse(currentDb))
+      validateDatabaseName(db)
```

Comment:
MINOR: So `formatDatabaseName` and `validateDatabaseName` are used alot together. Shouldn't they be in the same method? Same goes for `formatTableName` and `validateTableName`.